QRCoder/BitmapByteQRCode.cs

```diff
@@ -119,22 +152,22 @@ private byte[] HexColorToByteArray(string colorString)
         return byteColor;
     }
 
+
     /// <summary>
-    /// Converts an integer to a 4-byte array.
+    /// Converts an integer to a 4 bytes and writes them to a byte array at given position
     /// </summary>
     /// <param name="inp">The integer to convert.</param>
-    /// <returns>Returns the integer as a 4-byte array.</returns>
-    private byte[] IntTo4Byte(int inp)
+    /// <param name="destinationIndex">Index of destinationArray where the converted bytes are written to</param>
+    /// <param name="destinationArray">Destination byte array that receives the bytes</param>
+    private void CopyIntAs4ByteToArray(int inp, int destinationIndex, ref byte[] destinationArray)
```

Comment:
```suggestion
    private void CopyIntAs4ByteToArray(int inp, int destinationIndex, byte[] destinationArray)
```
No need for the destination to be a `ref` type.  The reference itself will not change; only the contents change.

Reply:
Fixed in: https://github.com/codebude/QRCoder/commit/86c2e47561ef35b56f4be8f70d93c2ffd5ef40c9